assets/js/test/unit/specs/components/vanilla_vue_input.spec.js

```diff
@@ -0,0 +1,21 @@
+import { createLocalVue, mount } from "@vue/test-utils"
+import VanillaVueInput from '@/components/VanillaVueInput.vue'
+
+const localVue = createLocalVue()
+
+describe('VanillaVueInput', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = mount(VanillaVueInput, {
+      localVue,
+      props: {
+        id: "Foo"
+      }
+    })
+  })
+
+  it('renders the correct markup', () => {
+    expect(wrapper.html()).toContain("<form>\n  <fieldset><input type=\"text\" placeholder=\"Type a value here...\"></fieldset>\n  <p>Value: </p>\n</form>")
+  })
+})
```

Comment:
Hey @jrgriffiniii until we get to #30 , mind installing a Prettier extension in your IDE, and set it to default settings?   Otherwise if we're both working on similar files the diffs are going to be a of of quotation marks and semicolons:)

Reply:
Also, just a thought, what about renaming the file using camel case to establish a pattern for .js files?   I know it's a small thing and no big deal, but I've found switching between lots of different projects, just quick eye glancing files, when I see camel case I know its JS, and snake case (is that what it's called?) I know it's generally Ruby, Elixir, etc.